source/Symbol/SwiftASTContext.cpp

```diff
@@ -6234,7 +6234,7 @@ VisitNodeQualifiedArchetype (SwiftASTContext *ast,
     {
         swift::Demangle::Node::iterator end = cur_node->end();
         VisitNodeResult type_result;
-        size_t index = LLDB_INVALID_ADDRESS;
+        uint64_t index = LLDB_INVALID_ADDRESS;
```

Comment:
As Todd Fiala stated, this is all generic stuff that should go directly into llvm.org, then it will be merged into swift-lldb by the auto merger.